### ISSUE TITLE
Unable to resolve path when using jsx-a11y as a plugin

### ISSUE THREAD

**Issue by dabit1** [evcohen/eslint-plugin-jsx-a11y#649]
Hi everyone,
I'm facing a problem when I add "jsx-a11y" in "plugins" in my .eslintrc. After renaming some files of my project (react components) eslint is throwing "Unable to resolve path...". If I restart my computer the error still. I only can delete "jsx-a11y" from eslintrc plugins in order to make my development server runs again.

Does anybody faced that error?

**Comment by ljharb** [comment]
Are any of your eslint things installed globally? eslint 6 forces them all to be installed locally (as they should always have been anyways).

**Comment by dabit1** [comment]
I have checked and they aren't.

**Comment by ljharb** [comment]
Do you have `eslint-plugin-jsx-a11y` installed? does `npm ls` exit zero?

**Comment by dabit1** [comment]
Yes it is installed. About `npm ls`, it exit zero because of some peer dependencies but actually they are installed, just in a lower version.

**Comment by ljharb** [comment]
if `npm ls` exits nonzero, then your dep graph is invalid, and you can't expect anything to work. Peer deps are required and the versions must match what's declared, always. If you fix that, does this problem go away?

**Comment by jessebeach** [comment]
@dabit1 try deleting your `node_modules` directory and reinstalling fresh with `npm install`. 

I don't believe this is an issue with this plugin. I'm going to close this issue until we get further details to work with.
